src/site/content/en/blog/layoutNG/index.md

```diff
@@ -0,0 +1,234 @@
+- **Writing Modes** <br>The legacy engine had many problems placing out of flow elements to default position in vertical writing modes. See the next section for more details around improved writing mode support.
+
+<br>
+
+### RTL & Writing Modes
+LayoutNG was designed from the ground up to support vertical writing modes and bi-directional content. This fixes numerous issues around both writing modes, right-to-left inlines, and orthogonal flows.
+
+#### Bidirectional Text
+LayoutNG supports the most up-to-date Unicode bidirectional algorithm defined by [The Unicode Standard](https://unicode.org/standard/standard.html). Not only does it fix various cases where rendering was not correct, it also includes missing features in the old engine such as paired bracket support ([#302469](https://crbug.com/302469)).
+
+#### Orthogonal Flows
+LayoutNG improves vertical flow layout correctness, for more correct positioning of absolute positioned objects, sizing of orthogonal flow boxes especially when percent is used, and so forth. Among the 1,258 tests in W3C test suites, **103 tests that failed in the old engine pass in LayoutNG.**
+
+#### Intrinsic Sizing
+Intrinsic sizes are now calculated correctly when a LayoutNG block contains children in an orthogonal writing mode.
+
+<br>
+
+### Text Layout & Line Breaking
+The old layout engine in Chromium does text layout element-by-element and line-by-line. This has historically worked well but requires a lot of extra complexity to support scripts and to achieve good performance. It's also prone to inconsistencies in measurements which tend to manifest themselves as subtle differences in sizing of size-to-content containers and their content or unnecessary line breaks.
+
+In LayoutNG, text layout is done on a paragraph level and is then split into lines. This allows for better performance, higher quality text rendering, and more consistent line breaking. The most notable differences here from a developer and user standpoint are detailed below.
+
+#### Joining across Element Boundaries
+In some scripts graphemes join with adjacent ones and changes presentation. In LayoutNG this works even if the graphemes are in different elements, allowing the joins to be preserved even if different styling is applied ([#6122](https://crbug.com/6122)).
+
+{% Aside 'key-term' %}
+  a grapheme is the smallest unit of a writing system of any given language
+{% endAside %}
+
+The example below shows the rendering of the following HTML in legacy layout and LayoutNG respectively:
+
+```html
+<div>&#1606;&#1587;&#1602;</div>
+<div>&#1606;&#1587;<span>&#1602;</span></div>
+```
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="legacy_shape.png" alt="proper grapheme on left and separated improper rendering on right">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 3a, Legacy Layout</small><br>
+      Note how the form of the second letter changes
```

Comment:
i think i kept this short intentionally so it laid out better as a caption. i'll review tomorrow 👍

Reply:
OK, I hear that. My version is definitely longer. I was going for precision: "changes" sounds (to me) like these are two steps in a sequence rather than two different pieces of code. And I wanted to get the idea that the difference is incorrect.  Basically, if folks are skimming, I wanted the figure to stand on its own as well as it could. But not a big deal, so feel free to disregard.